code/modules/clothing/shoes/magboots.dm

```diff
@@ -120,6 +120,14 @@
 	icon_state = "capboots0"
 	base_state = "capboots"
 	anchoring_system_examine = "Its anchoring spikes appear to be"
+	
+//Magnificent
+/obj/item/clothing/shoes/magboots/magnificent
+	desc = "The secret meaning of what mag stands for."
+	name = "magnificent mag boots"
+	icon_state = "MAGNIFICENTboots0"
+	base_state = "MAGNIFICENTboots"
+	anchoring_system_examine = "Impressive!"
```

Comment:
Do you know what the purpose of this var is?
Currently when examined, the player will see something like `Impressive! enabled.` at the end of the description, which doesn't make sense to me.

Reply:
Shh I totally didn't edit that while distracted.